src/clincoded/static/components/variant_curation.js

```diff
@@ -606,7 +606,7 @@ class VariantAddHistory extends Component {
 
         return (
             <div>
-                <span>Variant <strong><a href={"/variant-central/?variant=" + variant.uuid}>{variant.clinvarVariantTitle ? variant.clinvarVariantTitle : (variant.hgvsNames.GRCh38 ? variant.hgvsNames.GRCh38 : variant.hgvsNames.GRCh37)}</a></strong> added</span>
+                <span>Variant <strong><a href={"/variant-central/?variant=" + variant.uuid}>{renderVariantTitle(variant)}</a></strong> added</span>
```

Comment:
It doesn't appear that `renderVariantTitle` has been defined/declared in this file.

Reply:
Good catch! Going to fix this.